contrib_bots/lib/googlesearch.py

```diff
@@ -0,0 +1,61 @@
+# See readme.md for instructions on running this code.
+from google import search
+
+def get_google_result(original_content):
+    original_content = original_content.split(' ', 1)[1]
+    try:
+        urls = search(original_content, stop=20)
+    except:
+        return "Error: search failed."
+
+    if not urls:
+        return "No URLs returned by google."
+
+    urls = list(urls)
```

Comment:
I convert to a list so that I can then choose the first returned search query to show to the user.